sitemap.dd

```diff
@@ -0,0 +1,203 @@
+Ddoc
```

Comment:
Yes, I really need some sleep :P
In any case, we should add this to the gitignore or even better: simply put it into `.generated`
https://github.com/dlang/dlang.org/pull/1725